### PR TITLE
Add Tag List Customer Resource Definition

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: pkg/apis/provider/v1alpha1/aws_tags.go [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "awstaglists.provider.giantswarm.io",
+		},
+		Spec: apiextensionsv1beta1.CustomResourceDefinitionSpec{
+			Group:   "provider.giantswarm.io",
+			Scope:   "Namespaced",
+			Version: "v1alpha1",
+			Names: apiextensionsv1beta1.CustomResourceDefinitionNames{
+				Kind:     "AWSTagList",
+				Plural:   "awstaglists",
```
Comment (pipo02mix) [inline pkg/apis/provider/v1alpha1/aws_tags.go]: Open to change the naming, did not have better idea

File: pkg/apis/provider/v1alpha1/aws_tags.go [hunk 2]
```diff
@@ -0,0 +1,80 @@
+package v1alpha1
+
+import (
+	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// NewAWSTagListCRD returns a new custom resource definition for AWSConfig. This
```
Comment (rossf7) [inline pkg/apis/provider/v1alpha1/aws_tags.go]:
```suggestion
// NewAWSTagListCRD returns a new custom resource definition for AWSTagList. This
```

**Comment by xh3b4sd** [comment]
Is this PR still valid? Can we merge or close?

**Comment by pipo02mix** [comment]
Not anymore, aws tag operator has been discontinued

**Pull request closed without merging**